repose-aggregator/core/repose-core/src/test/resources/log4j2-test.xml

```diff
@@ -16,6 +16,7 @@
         <Logger name="org.apache" level="warn"/>
         <Logger name="org.eclipse.jetty" level="off"/>
         <Logger name="org.openrepose" level="debug"/>
+        <Logger name="org.openrepose.powerfilter" level="trace"/>
```

Comment:
Not sure you meant to sneak this in, i'm also not really worried about it

Reply:
There are a few of the checks in the unit tests that were looking for logs that were only logged at the `trace` level just to make sure the right outcome was obtained using the correct path.

Reply:
Yeah, i saw that later.